files/docker-image-prune.timer

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: 2025 Stefan Haun <tux@netz39.de>
+# SPDX-License-Identifier: MIT
+
+[Unit]
+Description=Timer to run Docker Prune
+
+[Timer]
+OnCalendar=weekly
```

Comment:
could/shouldn't the interval be a role parammeter instead of a hard coded value?